consultation_analyser/support_console/views/consultations.py

```diff
@@ -34,54 +33,25 @@ def import_consultation_job(
     )
 
 
-@job("default", timeout=900)
-def delete_consultation_job(consultation: models.Consultation):
-    from django.db import connection, transaction
-
-    consultation_id = consultation.id
-    consultation_title = consultation.title
+def delete_consultation_job(consultation_id: UUID):
 
     try:
-        # Close any existing connections to start fresh
```

Comment:
by passing the consultation_id by value rather than by reference we can avoid this issue